src/components/Header/Header.test.tsx

```diff
@@ -1,125 +1,138 @@
 import React from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
-import { shallow } from 'enzyme';
+import { render, fireEvent, waitForElementToBeRemoved, waitForElement } from '@testing-library/react';
 
 import Header from './Header';
 
+const headerProps = {
+  username: 'verddacio-user',
+  scope: 'test scope',
+  withoutSearch: true,
+  handleToggleLoginModal: jest.fn(),
+  handleLogout: jest.fn(),
+};
+
+/* eslint-disable react/jsx-no-bind*/
 describe('<Header /> component with logged in state', () => {
-  let wrapper;
-  let routerWrapper;
-  let instance;
-  let props;
-
-  beforeEach(() => {
-    props = {
-      username: 'test user',
-      handleLogout: jest.fn(),
-      logo: '',
-      onToggleLoginModal: jest.fn(),
-      scope: 'test scope',
-      withoutSearch: true,
-    };
-    routerWrapper = shallow(
+  test('should load the component in logged out state', () => {
+    const { container, queryByTestId, getByText } = render(
+      <Router>
+        <Header onLogout={headerProps.handleLogout} onToggleLoginModal={headerProps.handleToggleLoginModal} scope={headerProps.scope} />
+      </Router>
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
+    expect(queryByTestId('header--menu-acountcircle')).toBeNull();
```

Comment:
Reminder, the test-id is fine for me, but, according the guidelines, should be the 3rd option. 

https://testing-library.com/docs/guide-which-query